components/database/couchdb-21/patches/02-couchup.patch

```diff
@@ -0,0 +1,8 @@
+--- apache-couchdb-2.1.0/rel/overlay/bin/couchup.orig	2017-10-07 13:09:47.591823720 +0000
++++ apache-couchdb-2.1.0/rel/overlay/bin/couchup	2017-10-07 13:11:25.777635346 +0000
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env python
++#!/usr/bin/amd64/python2.7
```

Comment:
Is it strictly necessary use 64-bit python here?

Reply:
No. The script communicates exclusively by the REST interface. No libraries from couchdb are used. I removed the patch.